downstream/modules/analytics/proc-top-template-job-explorer.adoc

```diff
@@ -9,7 +9,7 @@ You can view job instances for top templates in a cluster to learn more about in
 
 .Procedure
 
-. Navigate to menu:Clusters[].
+. From the navigation panel, select {MenuAAClusters}.
 . Select a cluster from the clusters drop-down list. The view will update with that cluster's data.
 . Click on a template name in *Top Templates*.
 . Click btn:[View all jobs] in the modal that appears.
```

Comment:
@dcdacosta In the output file, the brackets around `View all jobs` still appears. Was this intentional?